Kernel/Term.hpp

```diff
@@ -171,7 +176,35 @@ class TermList {
   /** make the term empty (so that isEmpty() returns true) */
   inline void makeEmpty()
   { _content = FUN; }
+  inline static TermList empty()
+  { TermList out; out.makeEmpty(); return out; }
   /** make the term into a reference */
```

Comment:
The comment above is now misplaced!

Also, could you please add a bit of comment explaining what `Top` is for?